extra/external-worker-aws-cf-template.yaml

```diff
@@ -300,7 +344,136 @@ Resources:
               Ref: VolumeSize
             VolumeType: "gp2"
             DeleteOnTermination: "true"
+      UserData:
+        Fn::Base64:
+          Fn::Join:
+          - ''
+          - - "#!/bin/bash -xe\n"
+
+            - "export STACK_NAME="
+            - Ref: AWS::StackId
+            - "\n"
+
+            - "export SCHEDULER_API_ADDRESS='"
+            - Ref: SchedulerApiAddress
+            - "'\n"
+
+            - "export SCHEDULER_HOST="
+            - Ref: SchedulerHost
+            - "\n"
+
+            - "export SCHEDULER_PORT="
+            - Ref: SchedulerPort
+            - "\n"
+
+            - "export TSA_PUBLIC_KEY='"
+            - Ref: TsaPublicKey
+            - "'\n"
+
+            - "export WORKER_KEY='"
+            - Ref: WorkerKey
+            - "'\n"
+
+            - "export TEAM_ID="
+            - Ref: TeamId
+            - "\n"
+
+            - |
+              export LOG_FILE="/var/log/user-data.log"
+              exec &> >(tee -a ${LOG_FILE})
+
+              function finish {
+                  if [ $? -eq 0 ]; then
+                      echo "cloudformation signal-resource SUCCESS"
+                      aws cloudformation signal-resource --stack-name ${STACK_NAME} --logical-resource-id WorkersGroup --unique-id ${AWS_UNIQUE_ID} --region ${AWS_DEFAULT_REGION} --status SUCCESS
+                  else
+                      set +e
+                      echo "cloudformation signal-resource FAILURE"
+                      if ! [ -f "/tmp/keeprunning" ]; then
+                          aws cloudformation signal-resource --stack-name ${STACK_NAME} --logical-resource-id WorkersGroup --unique-id ${AWS_UNIQUE_ID} --region ${AWS_DEFAULT_REGION} --status FAILURE
+                          sleep 60
+                          halt -p
+                      fi
+                  fi
+              }
+              trap finish EXIT
+              echo "### starting setup of cycloid worker"
+              apt-get update
+              apt-get install -y git python-setuptools curl jq
+              easy_install pip
+              pip install -U cryptography
+              pip install ansible==2.4.3
+              pip install awscli
 
+              # Be able to use paris region (https://github.com/boto/boto/issues/3783)
+              pip install --upgrade boto
+
+              echo '[Boto]
+              use_endpoint_heuristics = True' > /etc/boto.cfg
```

Comment:
Do we really want to erase the file instead of adding / setting this config?

Reply:
I can open an issue for this. I only touched the UserData for telegraf installation.

Reply:
issue #29